src/user-statuses/userStatusTextSuggestions.js

```diff
@@ -1,11 +1,19 @@
 /* @flow strict-local */
 
+type Item = [
+  $ReadOnly<{| emoji_name: string, emoji_code: string, reaction_type: 'unicode_emoji' |}>,
+  string,
+];
+
 // This is a separate file due to Prettier having issues with formatting unicode
 // Using `prettier-ignore` does not help.
```

Comment:
Hmm, seems like this comment stops applying.

If so, we can also fold this file into `UserStatusScreen.js`.